### PR TITLE
Basic Database Operation & DB connection

### PR DESCRIPTION
**Pull request by yangfan2001**
# Pull Request Template

## Description

Add basic database operation for the project.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

## Note:
Include the information the reviewers need to know.

### CODE REVIEW

**Review by zyf1229 — APPROVED** [review]
LGTM!

**Comment by Yanglin-Tao** [comment]
Can you please add more documentation regarding the RDS database connection? Thanks

**Comment by yangfan2001** [comment]
> Can you please add more documentation regarding the RDS database connection? Thanks

Solved! https://github.com/nyuoss/glaceon-google-suite-tasks/pull/13

File: src/open_source_python_template/db/db.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+from psycopg2 import pool
+from datetime import datetime, timezone
+
+
+class DBConfig:
+    DATABASE_HOST = ""
+    DATABASE_NAME = "task-db"
+    DATABASE_USER = "postgres"
+    DATABASE_PASSWORD = ""
```
Comment (zyf1229) [inline src/open_source_python_template/db/db.py:9]: It is kind of insecure for host and password to be hardcoded directly into the codebase.

**Pull request merged**